tutorial/.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,31 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the README at:
+// https://github.com/microsoft/vscode-dev-containers/tree/v0.187.0/containers/alpine
+{
+	"name": "cc65 Assembler",
+	"build": {
+		"dockerfile": "Dockerfile",
+		// Update 'VARIANT' to pick an Alpine version: 3.10, 3.11, 3.12, 3.13
+		"args": { "VARIANT": "3.13" }
+	},
+
+	// Set *default* container specific settings.json values on container create.
+	"settings": {},
+
+	// Add the IDs of extensions you want installed when the container is created.
+	// Note that some extensions may not work in Alpine Linux. See https://aka.ms/vscode-remote/linux.
+	"extensions": [
+		"tlgkccampbell.code-ca65"
```

Comment:
We can install extensions that work server side, in this case the cc65 syntax and problem matchers.

Reply:
I would skip this extension. The point is to allow for VASM compilation as well, and these problem matchers are pretty specific.
This is one problem. The other is that the paths reported by CC65 don't match those of the workspace, making the problem matcher pretty useless. When I do find proper solution for this I would stick to regex-based matcher, it should be sufficient.